pyiron_base/job/generic.py

```diff
@@ -1514,6 +1516,10 @@ def _type_from_hdf(self):
                 self.executable.version = self._hdf5["VERSION"]
             except ValueError:
                 self.executable.executable_path = self._hdf5["VERSION"]
+            try:
+                self.executable.additional_arguments = self._hdf5["RUNARGS"]
```

Comment:
rather than `try except` I would prefer `if "RUNARGS" in self._hdf5.list_nodes():`